src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+@use '../_variables/variables' as *;
+
+.page {
+  font-family: $PrimaryFont;
```

Comment:
The variable `$PrimaryFont` should be in lowercase according to the code style rules. Please rename it to `$primary-font`.